src/tree/mod.rs

```diff
@@ -487,7 +485,7 @@ impl RatchetTree {
         for (path_secret, copath_node) in path_secrets.iter().zip(copath.iter()) {
             let node_ciphertexts: Vec<HpkeCiphertext> = self
                 .resolve(*copath_node)
-                .par_iter()
+                .iter()
```

Comment:
Why is the parallelization removed here?

Reply:
Because extensions are a trait now that doesn't implement [Sync](https://doc.rust-lang.org/std/marker/trait.Sync.html) yet. We can add that later. But I don't think this is necessary right now. I filed #108